src/olympia/constants/base.py

```diff
@@ -329,20 +329,15 @@
 ADDON_UUID = r'(?P<uuid>[\w]{8}-[\w]{4}-[\w]{4}-[\w]{4}-[\w]{12})'
 
 # Default strict_min_version and strict_max_version for WebExtensions
-DEFAULT_WEBEXT_MIN_VERSION = '42.0'
+# SHA256 is supported with versions >=58.0
+DEFAULT_WEBEXT_MIN_VERSION = '58.0'
 DEFAULT_WEBEXT_MAX_VERSION = '*'
 
 # Android only started to support WebExtensions with version 48
-DEFAULT_WEBEXT_MIN_VERSION_ANDROID = '48.0'
-
-# The default version of Firefox that supports WebExtensions without an id
-DEFAULT_WEBEXT_MIN_VERSION_NO_ID = '48.0'
-
-# The default version of Firefox that supported `browser_specific_settings`
-DEFAULT_WEBEXT_MIN_VERSION_BROWSER_SPECIFIC = '48.0'
+DEFAULT_WEBEXT_MIN_VERSION_ANDROID = '58.0'
 
 # The version of desktop Firefox that first supported static themes.
```

Comment:
Not sure what the ask is here.

Reply:
```suggestion
# The version of desktop Firefox that first supported static themes is 53.0,
# but as above because we're now signing with SHA256, the minimum version has
# to be 58.0.
```